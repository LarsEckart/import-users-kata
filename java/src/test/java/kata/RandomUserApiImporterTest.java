package kata;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

import java.util.ArrayList;

class RandomUserApiImporterTest {

    @Test
    void applesauce() {
        var apiImporter = new RandomUserApiImporter("https://randomuser.me/api/?inc=gender,name,email,location,dob&results=2&seed=a9b25cd955e2333h");

        ArrayList<User> users = apiImporter.importUsers();

        Approvals.verifyAll("Imported Users", users);
    }
}
