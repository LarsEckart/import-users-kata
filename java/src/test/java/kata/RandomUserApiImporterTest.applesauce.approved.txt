Imported Users[0] = 100000000001,Mathys Marie,1998-07-19,France,66860,mathys.marie@example.com
Imported Users[1] = 100000000002,Ceyhun Yazıcı,1977-01-21,Turkey,16330,ceyhun.yazici@example.com
